src/main/scala/com/touk/hiring_task/multiplex/dto/CinemaDto.scala

```diff
@@ -0,0 +1,14 @@
+package com.touk.hiring_task.multiplex.dto
+
+import com.touk.hiring_task.multiplex.model.Place
+import scala.language.implicitConversions
+
+case class PaymentDto(amount: Double, expireAt: Long)
```

Comment:
do you think Double is a good type for the amount?

Reply:
for the amount the best is BigDecimal, but in this case when we have simple calculation I decided to use Double :/